packages/terra-tabs/src/common-tabs/_Tabs.jsx

```diff
@@ -59,22 +67,52 @@ const propTypes = {
    */
   onChange: PropTypes.func,
   /**
-   * Whether or not the tab is draggable.
+   * Callback function when add button selection has changed.
+   * Parameters: 1. Event 2. Selected pane's key
    */
+  onSelectAddButton: PropTypes.func,
+  /**
+   * Callback function when a tab is closing.
+   * It receives two parameters: 1. Item key of the tab to close. 2. MetaData associated with the tab.
+   */
+  onClosingTab: PropTypes.func,
+
+  /**
+   * Indicates whether the tabs are closable, allowing the user to close tabs.
+   */
+  isClosable: PropTypes.bool.isRequired,
+
+  /**
+   * Callback function when a tab is closed.
+   * It receives two parameters: 1. Item key of the closed tab. 2. MetaData associated with the closed tab.
+   */
+  onClose: PropTypes.func,
```

Comment:
onClosingTab and onClose are same remove 1 of them

Reply:
This is used with this behaviour please check this comment (https://github.com/cerner/terra-framework/pull/1738#discussion_r1289081957)

Reply:
Done https://github.com/cerner/terra-framework/commit/ca07c6755741f69957198bcb28e0ed3fbc6ffb15